src/main/webapp/dataset-license-terms.xhtml

```diff
@@ -237,7 +237,8 @@
                 </p:fragment>
             </div>
         </div>
-       <div class="panel panel-default" jsf:rendered="#{!empty editMode or empty editMode and ((!empty termsOfUseAndAccess.termsOfAccess and DatasetPage.restrictedFileCount > 0) 
+       <div class="panel panel-default" jsf:rendered="#{!empty editMode or empty editMode and (DatasetPage.restrictedFileCount > 0 
```

Comment:
@sekmiller  Does this even need the first two checks for edit mode since it seems they'd cancel each other out? If that goes away so can a parenthesis layer, I think, making it a little cleaner)

Reply:
Ok. that makes sense. In my internal parser I was saying, "you want to render this if you're in edit mode or you're not in edit mode but at least one of the fields is filled in". I'll change it.

Reply:
Ah, you're right - I missed that it was an AND and not an OR. Either way, removing it is simpler (since `empty editMode` has to be true if `!empty editMode` is false)